comments.php

```diff
@@ -23,6 +23,15 @@
 <div id="comments" class="comments-area">
 
 	<?php
+	// Hide pingbacks if they are disabled
+	if (! pings_open() ) : ?>
+		<style>
```

Comment:
This looks a bit nasty. Not keen on inline style.

You can filter the classes that are applied to the <body> tag in WordPress. We could add a .pingbacks--off type class may be (I think as default there's ones about comments open and closed) then we can add a class for pingbacks more like:

`.pingbacks--off .pingbacks { display:none;}`

That is, if we can't remove the pingbacks in code and are forced to use display:none. Which in itself feels a bit hacky.